### PR TITLE
Runs additional hooks on `customize apply`

### LINKED ISSUE #44: Cluster customizer: should `start.d` scripts be run on apply?
When installing a customization profile post-startup, the `configure.d` and `member-join.d` hooks in that profile are called in turn (where present).

Some profiles also have `start.d` hooks, but these are not called.

This means that, for example, when applying `feature/configure-docker` the user will manually need to `sudo systemctl start docker` before being able to use Docker (since the `start.d` script which otherwise does so is not called).

Should we also be triggering a profile's `start.d` hooks when applying post-startup?

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
`alces customize apply` runs the `start` and `node-start` in addition to the `configure` hook
fixes #44

### CODE REVIEW

**Comment by WilliamMcCumstie** [comment]
Needs to update change log when merged

File: clusterware-customize/lib/functions/customize-repository.functions.sh [hunk 1]
```diff
@@ -244,6 +244,8 @@ customize_repository_apply() {
       if [[ "$is_preinit" != "preinit" ]]; then
         echo "Running configure for $profile_name"
         customize_run_hooks "configure:$repo_name-$profile_name"
+        customize_run_hooks "start:$repo_name-$profile_name"
+        customize_run_hooks "node-start:$repo_name-$profile_name"
```
Comment (mjtko) [inline clusterware-customize/lib/functions/customize-repository.functions.sh]: This should be `node-started`.

The `echo` line should probably be modified to say "Running event hooks for..." or something to match the new behaviour.

**Comment by WilliamMcCumstie** [comment]
Updated the message

**Comment by mjtko** [comment]
:+1:

**Pull request merged**